fhir-parent/pom.xml

```diff
@@ -74,7 +77,7 @@
             <dependency>
                 <groupId>${project.groupId}</groupId>
                 <artifactId>fhir-examples</artifactId>
-                <version>4.0.1-SNAPSHOT</version>
+                <version>${fhir-examples.properties}</version>
```

Comment:
not sure what we gain from this change, is it used anywhere else?

Reply:
it makes the version change very explicit